src/services/api/index.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+'use strict'
+
+import { postRequest } from '../fetch'
+import OneBlinkAppsError from '../errors/oneBlinkAppsError'
+
+// FNs
+import { downloadDraftData, uploadDraftData, putDrafts } from './drafts'
+import { uploadPreFillFormData, downloadPreFillFormData } from './prefill'
+import {
+  generatePaymentConfiguration,
+  acknowledgeCPPayTransaction,
+  verifyPaymentTransaction,
+} from './payment'
```

Comment:
This file will be internal, no need to re-export this all these functions

Reply:
Just did it so when we are consuming it we can import them all from the same place, as they were before, code is just more organised now